bin/entry.sh

```diff
@@ -6,13 +6,4 @@ else
     RUNTIME_CMD="$(which python)"
 fi
 
-## IMPORTANT: NEW RELIC CONFIGURATION -- WE ALWAYS WANT TO USE APM MODE!
-# Force classic New Relic agent mode for Lambda so data flows to APM instead of the serverless extension
-unset NEW_RELIC_LAMBDA_EXTENSION_ENABLED
-unset NEW_RELIC_LAMBDA_HANDLER
-unset NEW_RELIC_EXTENSION_LOGS_ENABLED
-unset NEW_RELIC_EXTENSION_SEND_FUNCTION_LOGS
-export NEW_RELIC_SERVERLESS_MODE_ENABLED=false
-export NEW_RELIC_CONFIG_FILE=${NEW_RELIC_CONFIG_FILE:-/app/newrelic.ini}
```

Comment:
I don't know if these env vars will be active after the merge, but that one above is key important, i.e. NEW_RELIC_CONFIG_FILE.

Reply:
ya, that's a legit point...

Reply:
but to fully cover our butts, added this one to the terraform list so it exists